src/components/forms/validator.ts

```diff
@@ -100,7 +102,7 @@ export const minMaxLength = (minLen, maxLen) => (value) =>
 
 export const ADDRESS_REPEATED_ERROR = 'Address already introduced'
 
-export const uniqueAddress = (addresses: string[]) =>
+export const uniqueAddress = (addresses: string[] | List<string>) =>
```

Comment:
can we add a return type here? and also for the returned function

Reply:
<img width="982" alt="Screenshot 2020-07-07 at 17 26 53" src="https://user-images.githubusercontent.com/16622558/86788308-12f66400-c077-11ea-8d36-fcc553427b5c.png">

please pay more attention to eslint warnings

Reply:
I'll skip this, as you're already working on it: https://github.com/gnosis/safe-react/pull/1108/files#diff-08207e76e4b0f60ae0de54059ac9474dR82